3_ProtectionProfile/BiocPP.adoc

```diff
@@ -129,16 +132,24 @@ Zero-effort Impostor Attempt::
 |0.9
 |5th August, 2019
 |Updates based on Public Review Draft 1 comments
+
+|0.9
+|5th December, 2019
+|Updates to make PAD optional
 |===
 
-== PP Introduction
+== PP-Module Introduction
+
+=== PP-Module Reference
+- PP-Module Reference: {doctitle}
+- PP-Module Version: {revnumber}
+- PP-Module Date: {revdate}
 
-=== PP Reference Identification
-- PP Reference: {doctitle}
-- PP Version: {revnumber}
-- PP Date: {revdate}
+=== Base PP identification
```

Comment:
Base-PPs should be defined in the PPC, not the PP-Module I would think, so this is one of the places where I have a problem with the MDFPP listing since it would imply I also need to list GPOS here as well.

Reply:
The reason why I add this "Base PP identification" is that CC Part1 requires PP-Module to include this identification. Please see "Figure 11 - PP-Module content" in CC part 1. Logically you are right but it may be better to follow CC.

*) https://www.commoncriteriaportal.org/files/ccfiles/CCPART1V3.1R5.pdf

Reply:
See my next comment about referencing